statement-table/src/generic.rs

```diff
@@ -267,40 +267,26 @@ impl<Ctx: Context> CandidateData<Ctx> {
 			Ctx::GroupId, Ctx::Candidate, Ctx::AuthorityId, Ctx::Signature,
 		>>
 	{
-		if self.can_be_included(validity_threshold) {
-			let validity_votes: Vec<_> = self.validity_votes.iter()
-				.filter_map(|(a, v)| match *v {
-					ValidityVote::Invalid(_) => None,
-
-					ValidityVote::Valid(ref s) =>
-						Some((a, ValidityAttestation::Explicit(s.clone()))),
-					ValidityVote::Issued(ref s) =>
-						Some((a, ValidityAttestation::Implicit(s.clone()))),
-				})
-				.take(validity_threshold)
```

Comment:
If someone comes up with a good explanation why we only want to have `validity_threshold` votes forwarded, please tell it. Otherwise, I don't see any reason to restrict this.

Reply:
legacy